source/connection-string/tests/valid-options.yml

```diff
@@ -15,3 +15,11 @@ tests:
             db: "admin"
         options:
             authmechanism: "MONGODB-CR"
+    -
+        description: "Missing delimiting slash between hosts and options"
+        uri: "mongodb://example.com?w=1"
+        valid: true
+        warning: ~
+        hosts: ~
+        auth: ~
+        options: ~
```

Comment:
Consider adding an assertion that options are respected when there is no delimiting slash (e.g. that the write concern is set here, or some other option if it's easier).

Reply:
Changed this to check for tls being set to true.